### PR TITLE
Refactor local dev environment to use docker for development

### PR DESCRIPTION
**Pull request by melatonin355**
- Create docker-compose.dev.yml to simplify local dev environment with hot-reload. It runs all migrations and starts development server. 

You can run and test it with: 

```
docker compose -f "docker-compose.dev.yml" up 
```

- Added http://localhost:8002/test/  endpoint 
- Modified / to go to custom index page

### CODE REVIEW

File: greysend_project/chat/views.py [hunk 1]
```diff
@@ -1,3 +1,10 @@
 from django.shortcuts import render
+from django.http import HttpResponse
 
 # Create your views here.
+def test_view(request):
+    return HttpResponse("This is a test endpoint!")
+
+
+def landing_page(request):
+    return render(request, 'landing_page.html')
```
Comment (kayernyc) [inline greysend_project/chat/views.py:10]: Super minor nit-pick. There should be a return here.

File: docker-compose.dev.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      POSTGRES_USER: admin
+      POSTGRES_PASSWORD: asdf1234
+      POSTGRES_HOST: postgres
+      PRODUCTION: false  # set to false for local development
+      DEBUG: true  # enable debug mode for local development
+    ports:
+      - 8002:8002
+    depends_on:
+      - postgres
+    volumes:
```
Comment (kayernyc) [inline docker-compose.dev.yml:21]: aha! Now I see how this works!

**Review by kayernyc — APPROVED** [review]
One tiny comment! Very nice work!

**Review by eclbg — APPROVED** [review]
LGTM!

**Pull request merged**